### PR TITLE
✨ Bump to k8s.io v1.24.0-rc.1

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 // Version returns the version of the main module
 func Version() string {
 	info, ok := debug.ReadBuildInfo()
-	if !ok {
+	if !ok || (info != nil && info.Main.Version == "") {
```
Comment (sbueringer) [inline pkg/version/version.go:26]: A unit test failed without this change

It looks like previously ok was false. Now we get ok==true + an info object with an empty Version

Comment (alvaroaleman) [reply pkg/version/version.go:26]: okay, interesting. That is IMHO a go bug as it breaks comoatibility.

Comment (alvaroaleman) [reply pkg/version/version.go:26]: Shouldn't it be
```suggestion
	if !ok || info == nil || info.Main.Version == "" {
```

**Review by alvaroaleman — APPROVED** [review]

File: pkg/version/version.go (hunk 1 above)
Comment (AlmogBaku) [reply pkg/version/version.go]: see #662 for explanation for this issue

**Pull request merged**